source/docs/software/commandbased/organizing-command-based.rst

```diff
@@ -365,9 +445,17 @@ If we wish to write composite commands as their own classes, we may write a cons
             );
         }
     }
+
   .. code-block:: c++
 
-    // TODO
+    class IntakeThenOuttake : public frc2::SequentialCommandGroup {
```

Comment:
CommandHelper is needed.

Reply:
is that as well as the frc2::SequentialCommandGroup?